debian/control

```diff
@@ -20,7 +15,7 @@ Description: Calculate in an elementary way.
 
 Package: pantheon-calculator
 Architecture: any
-Depends: io.elementary.calculator, ${misc:Depends}
```

Comment:
This line is actually for the transitional package, so we want to keep it as pulling in `io.elementary.calculator`. The runtime depends we want to change are on line 16